src/main/scala/org.constellation.cmhotwallet/model/request/CLTransaction.scala

```diff
@@ -0,0 +1,49 @@
+package org.constellation.cmhotwallet.model.request
+
+import io.circe.{Decoder, Encoder}
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+
+object CLTransaction {
+  import org.constellation.wallet.{
+    Edge,
+    HashSignature,
+    Id,
+    LastTransactionRef,
+    ObservationEdge,
+    SignatureBatch,
+    SignedObservationEdge,
+    Transaction,
+    TransactionEdgeData,
+    TypedEdgeHash
+  }
+
+  implicit val clLastTransactionRefEncoder: Encoder[LastTransactionRef] = deriveEncoder
+  implicit val clLastTransactionRefDecoder: Decoder[LastTransactionRef] = deriveDecoder
```

Comment:
Why it's needed there? I believe it's specified in the `org.constellation.wallet` as well

Reply:
The wallet itself didn't send the transaction so it didn't need the encoders for http client - it used circe auto derivation to export the transaction to file and read from file.